Assets/Scripts/UI/InGame/Graphics/QualitySwitcher.cs

```diff
@@ -8,12 +8,21 @@ public class QualitySwitcher : UiElement {
 		public Text Label;
 		public Slider Slider;
 
-		public void UpdateQuality()
+        private void Start()
```

Comment:
The indentation of this class seems messed up somehow. So please format the document (Visual Studio: Ctrl+K, D )